pkg/kyverno/common/fetch.go

```diff
@@ -272,6 +272,33 @@ func convertResourceToUnstructured(resourceYaml []byte) (*unstructured.Unstructu
 	return resource, nil
 }
 
+// GetPatchedResource converts raw bytes to unstructured object
+func GetPatchedResource(patchResourceBytes []byte) (patchedResource unstructured.Unstructured, err error) {
+	var getErrString string
```

Comment:
Can we just call this `ErrString` instead of `getErrString` ?

Reply:
> 
> 
> Can we just call this `ErrString` instead of `getErrString` ?

yes, we can call it, np.